docs/ROADMAP.md

```diff
@@ -0,0 +1,81 @@
+# Purpose of PASS and MVP Outline
+
+## Project Goals
+
+The ultimate purpose of this project is to create a decentralized, federated system that puts users in control of their own data. Primarily home-insecure individuals who are most at risk of losing their important documents. We have validated through the Portland State University Homelessness Research & Action Collaborative (HRAC) that keeping important documents safe and available is a major problem of the housing insecure community and there are no good offerings today. 
```

Comment:
Not vital but is there a reason to use both "home-insecure" and "housing insecure"? Really just semantics but perhaps sticking with one would be helpful.

Reply:
This was copy and pasted from the original documentation. I can update terminology in a future fix to keep consistency.